living_documentation_generator/utils/utils.py

```diff
@@ -72,6 +72,25 @@ def make_absolute_path(path: str) -> str:
     return os.path.abspath(path)
 
 
+def validate_query_format(query_string, expected_placeholders) -> None:
+    """
+    Validate the placeholders in the query string.
+    Check if all the expected placeholders are present in the query and exit if not.
+
+    @param query_string: The query string to validate.
+    @param expected_placeholders: The set of expected placeholders in the query.
+    @return: None
+    """
+    actual_placeholders = set(re.findall(r"\{(\w+)\}", query_string))
+    missing = expected_placeholders - actual_placeholders
+    extra = actual_placeholders - expected_placeholders
+    if missing or extra:
```

Comment:
We should split these log messages by missing or extra. How often it happens together?

Reply:
Now, when this happen than this can be an output:
`ERROR - Missing placeholders: set(), Extra placeholders: {'another'}.`

Does `set()` looks ok to you?

Reply:
After your suggestion, I changed the logic. See commit: [8327094](https://github.com/AbsaOSS/living-doc-generator/pull/41/commits/83270948338e967b4bcc0951434e15482a420b48).